candi/bashible/bundles/debian/all/000_install_ca_certificates.sh.tpl

```diff
@@ -23,11 +23,17 @@ fi
 
 {{- if .registry.ca }}
 bb-event-on 'registry-ca-changed' '_update_ca_certificates'
-function _update_ca_certificates() {
+_update_ca_certificates() {
+  bb-flag-set containerd-need-restart
   update-ca-certificates
 }
 
 bb-sync-file /usr/local/share/ca-certificates/registry-ca.crt - registry-ca-changed << "EOF"
```

Comment:
```suggestion
bb-sync-file /usr/local/share/ca-certificates/d8-registry-ca.crt - registry-ca-changed << "EOF"
```
(duplicate) we should have done this before.

Reply:
This leads to undesirable work on the nodes.